styleguide/source/_patterns/02-molecules/tools-learn-more.twig

```diff
@@ -0,0 +1,12 @@
+<div class="ama__tools--learn-more">
```

Comment:
It looks like `--learn-more` is a sub element of `ama__tools` and as such should be denoted with a double underline instead of a double hyphen.